cmd/metal-api/internal/service/v1/switch.go

```diff
@@ -81,9 +89,11 @@ func NewSwitchResponse(s *metal.Switch, p *metal.Partition, nics SwitchNics, con
 		SwitchBase: SwitchBase{
 			RackID: s.RackID,
 		},
-		Nics:        nics,
-		Partition:   *NewPartitionResponse(p),
-		Connections: cons,
+		Nics:          nics,
+		Partition:     *NewPartitionResponse(p),
+		Connections:   cons,
+		LastSync:      s.LastSync,
```

Comment:
@majst01 here the complete sync struct is passed to the SwitchResponse

Reply:
stupid me